src/svg_avatar_builder.rs

```diff
@@ -166,7 +166,7 @@ impl SvgAvatarBuilder {
     }
 
     fn create_document() -> Document {
-        Document::new().set("viewBox", (-1.1, -1.1, 2.3, 2.3))
+        Document::new().set("viewBox", (-1, -1, 2, 2))
```

Comment:
Hey @kindlychung,

thanks for the PR. I did initially create the `viewBox` without the padding, but then stroke lines get cut off:

![image](https://github.com/nathiss/svg_avatars/assets/11808334/d851c3ef-d720-4bb0-a378-0962fe38395f)

I was trying to find a solution and it seems like allowing overflow to be visible on the `svg` tag fixes it.

```rust
fn create_document() -> Document {
    Document::new()
        .set("viewBox", (-1, -1, 2, 2))
        .set("overflow", "visible")
}
```

What do you think about it? If it coves your use case, then I'll be happy to merge the changes for you!

Reply:
Complete fine with it, thanks for looking into it!